server/src/test/java/de/uftos/controller/StudentControllerTest.java

```diff
@@ -0,0 +1,154 @@
+package de.uftos.controller;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+
+import io.restassured.http.ContentType;
+import java.util.List;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class StudentControllerTest {
+
+  static String karenId;
+  static String jasonId;
+  static String tagId;
+
+  @BeforeAll
+  static void createTestStudents() {
+    tagId = given().contentType(ContentType.JSON)
+        .body("""
+                    {
+                      "tagName": "Sehebehinderung"
+                    }""")
+        .when()
```

Comment:
Instead of having incredibly ugly multiline strings, can we use some JSON builder?

Reply:
done